From 37170d94b30195a2ef1a4ff060205eeaf8648a3d Mon Sep 17 00:00:00 2001
From: Steve Arnold <stephen.arnold42@gmail.com>
Date: Sat, 20 Aug 2016 15:22:50 -0700
Subject: [PATCH 2/2] ARM configs: add defconfig for bpi-r1

Note: bpi-r1 is *not* lamobo-r1, as it *requires* GMAC_TX_DELAY=3
instead of 4 (like lamobo) so ethernet can actually work.

Signed-off-by: Steve Arnold <nerdboy@gentoo.org>
---
 configs/Sinovoip_BPI_R1_defconfig | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)
 create mode 100755 configs/Sinovoip_BPI_R1_defconfig

diff --git a/configs/Sinovoip_BPI_R1_defconfig b/configs/Sinovoip_BPI_R1_defconfig
new file mode 100755
index 0000000..ca3b9bb
--- /dev/null
+++ b/configs/Sinovoip_BPI_R1_defconfig
@@ -0,0 +1,19 @@
+CONFIG_ARM=y
+CONFIG_ARCH_SUNXI=y
+CONFIG_MACH_SUN7I=y
+CONFIG_DRAM_CLK=432
+CONFIG_GMAC_TX_DELAY=3
+CONFIG_DEFAULT_DEVICE_TREE="sun7i-a20-sinovoip-bpi-r1"
+# CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
+CONFIG_SPL=y
+CONFIG_SYS_EXTRA_OPTIONS="AXP209_POWER,SUNXI_GMAC,RGMII,MACPWR=SUNXI_GPH(23),AHCI,USB_EHCI"
+# CONFIG_CMD_IMLS is not set
+# CONFIG_CMD_FLASH is not set
+# CONFIG_CMD_FPGA is not set
+CONFIG_DM_ETH=y
+CONFIG_PHYLIB=y
+# CONFIG_RTL8211X_PHY_FORCE_MASTER=y
+CONFIG_ETH_DESIGNWARE=y
+CONFIG_DM_SERIAL=y
+CONFIG_USB=y
+CONFIG_DM_USB=y
-- 
2.8.1

